packages/foundation/src/elements/navigation-rail-item.css

```diff
@@ -0,0 +1,75 @@
+.hap-navigation-rail-item {
+  font-family: var(--hap-typography-font-family-body);
+  width: 80px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: var(--hap-spacing-xs);
+  align-self: flex-start;
+  gap: var(--hap-spacing-xs);
+  cursor: pointer;
+  text-decoration: none;
+
+  .hap-navigation-rail-item-icon {
+    color: var(--hap-color-icon-secondary-on-light);
+    position: relative;
+
+    .hap-indicator {
+      position: absolute;
+      top: -1px;
+      right: -10px;
+
+      &.small {
+        top: 0;
+        right: calc((-1) * var(--hap-spacing-xs));
+      }
+    }
+  }
+
+  .hap-navigation-rail-item-label {
+    color: var(--hap-color-text-secondary-on-light);
+    font-size: var(--hap-typography-font-size-bodytext-xs);
+    font-weight: var(--hap-typography-font-weight-medium);
+  }
+
+  &:focus-visible {
+    outline: 2px solid var(--hap-color-border-focused-default);
+    background: none !important;
```

Comment:
Wenn der Fokus-State in der definition zum Schluss kommt, müsste das '!important" wegfallen können.